features/nanostack/mbed-mesh-api/source/WisunBorderRouter.cpp

```diff
@@ -186,3 +245,115 @@ int WisunBorderRouter::routing_table_get(ws_br_route_info_t *table_ptr, uint16_t
 
     return ws_bbr_routing_table_get(_mesh_if_id, (bbr_route_info_t *)table_ptr, table_len);
 }
+
+mesh_error_t WisunBorderRouter::set_radius_server_ipv6_address(const char *address)
+{
+    if (address) {
+        uint8_t ipv6_addr[16];
+        if (!stoip6(address, strlen(address), ipv6_addr)) {
+            return MESH_ERROR_PARAM;
+        }
+        // Stored address (returned by get) is in the format given by user of the interface
+        strcpy(_radius_ipv6_addr, address);
+        _radius_ipv6_addr_set = true;
+    } else {
+        _radius_ipv6_addr_set = false;
+    }
+
+    return set_bbr_radius_address();
+}
+
+mesh_error_t WisunBorderRouter::get_radius_server_ipv6_address(char *address)
+{
+    if (!_radius_ipv6_addr_set) {
```

Comment:
You would not need these booleans if you would always read these variables from the Mbed json in the begining?

Reply:
The RADIUS configuration could be unspecified in the .json (use internal TLS) and then later enabled by configuration function calls.